WcaOnRails/db/migrate/20160207125654_add_friendly_id_to_competitions.rb

```diff
@@ -0,0 +1,12 @@
+class AddFriendlyIdToCompetitions < ActiveRecord::Migration
+  def up
+    add_column :Competitions, :friendly_id, :string, null: false, default: ""
```

Comment:
I guess so.
Em 31/03/2016 19:41, "Jeremy Fleischman" <notifications@github.com>
escreveu:

> In WcaOnRails/db/migrate/20160207125654_add_friendly_id_to_competitions.rb
> <https://github.com/cubing/worldcubeassociation.org/pull/381#discussion_r58139669>
> :
>
> > @@ -0,0 +1,12 @@
> > +class AddFriendlyIdToCompetitions < ActiveRecord::Migration
> > +  def up
> > +    add_column :Competitions, :friendly_id, :string, null: false, default: ""
>
> Should we have a unique constraint on friendly_id?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/cubing/worldcubeassociation.org/pull/381/files/e5c12db34efa24d63b37c5b6945cd530603f3180#r58139669>
>